src/components/Footer/index.js

```diff
@@ -109,7 +108,7 @@ const Footer = () => {
             <a href="/">
               <img
                 alt="The Debt Collective"
-                src={logo}
+                src="/img/logo-dark.svg"
```

Comment:
@duranmla why was this changed?

Reply:
https://github.com/debtcollective/student-debt-campaign-netlify/pull/23/commits/9b97d3cbc07fda3845dedc3eedc142202d6faa83 basically I have an issue with the file-mock that didn't allow me to run my tests on jest using the `--watch` mode.

This file https://github.com/gatsbyjs/gatsby/blob/master/examples/using-jest/__mocks__/file-mock.js is created to say to jest here https://github.com/gatsbyjs/gatsby/blob/master/examples/using-jest/jest.config.js#L7 that we need to "ignore" some assets such as svg BUT placing those files outside the static folder  make this to not work even when we used to have https://github.com/debtcollective/student-debt-campaign-netlify/pull/23/commits/927ea4fce4236eb078de07a49a0d785d8543a126

**TL;DR** it was a workaround to avoid to have the issue I had with jest, since the configuration was done but not working for a reason that I couldn't find.